packages/twenty-server/src/workspace/workspace-query-runner/jobs/call-webhook-jobs.job.ts

```diff
@@ -20,7 +21,7 @@ export enum CallWebhookJobsJobOperation {
 
 export type CallWebhookJobsJobData = {
   workspaceId: string;
-  objectNameSingular: string;
+  objectMetadataItem: ObjectMetadataInterface;
```

Comment:
@charlesBochet I rather just call it `objectMetadata`, it's called like that in other part of the code base